source/game/kart/KartMove.cc

```diff
@@ -1275,18 +1278,14 @@ f32 KartMove::calcVehicleAcceleration() const {
         return 1.0f;
     }
 
-    std::vector<f32> as;
-    std::vector<f32> ts;
+    std::span<const f32> as;
+    std::span<const f32> ts;
     if (state()->isDrifting()) {
-        const auto &as_arr = param()->stats().accelerationDriftA;
-        const auto &ts_arr = param()->stats().accelerationDriftT;
-        as = {as_arr.begin(), as_arr.end()};
-        ts = {ts_arr.begin(), ts_arr.end()};
+        as = param()->stats().accelerationDriftA;
```

Comment:
Could remove the if-statement here if you do this
```suggestion
        auto &stats = param()->stats();
        bool isDrifting = state()->isDrifting();
        as = isDrifting ? stats.accelerationDriftA : stats.accelerationStandardA;
        ts = isDrifting ? stats.accelerationDriftT : stats.accelerationStandardT;
```

Reply:
This does not work, as the operands for instantiation are different types.